collector/go.mod

```diff
@@ -110,15 +110,15 @@ require (
 	github.com/tklauser/numcpus v0.6.0 // indirect
 	github.com/yusufpapurcu/wmi v1.2.2 // indirect
 	go.opencensus.io v0.24.0 // indirect
-	go.opentelemetry.io/collector/consumer v0.68.0 // indirect
-	go.opentelemetry.io/collector/exporter/loggingexporter v0.68.0 // indirect
-	go.opentelemetry.io/collector/exporter/otlpexporter v0.68.0 // indirect
-	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.68.0 // indirect
-	go.opentelemetry.io/collector/featuregate v0.68.0 // indirect
-	go.opentelemetry.io/collector/pdata v1.0.0-rc2 // indirect
-	go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.68.0 // indirect
-	go.opentelemetry.io/collector/receiver/otlpreceiver v0.68.0 // indirect
-	go.opentelemetry.io/collector/semconv v0.68.0 // indirect
+	go.opentelemetry.io/collector/consumer v0.69.1 // indirect
+	go.opentelemetry.io/collector/exporter/loggingexporter v0.69.1 // indirect
+	go.opentelemetry.io/collector/exporter/otlpexporter v0.69.1 // indirect
+	go.opentelemetry.io/collector/exporter/otlphttpexporter v0.69.1 // indirect
+	go.opentelemetry.io/collector/featuregate v0.69.1 // indirect
+	go.opentelemetry.io/collector/pdata v1.0.0-rc3.0.20230109164642-7d168dd20efd // indirect
```

Comment:
```suggestion
	go.opentelemetry.io/collector/pdata v1.0.0-rc3 // indirect
```

This seems wrong.

Reply:
It looks like the contrib v0.69.0 modules were released depending on v1.0.0-rc3.0.20230109164642-7d168dd20efd :(

Reply:
I think we've taken steps to avoid that in the future and I'm fine merging this since it shouldn't have negative impact and the collector releases used the same version of the pdata module.